src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -144,7 +144,35 @@ export const selectPendingAppointments = createSelector(
 export const selectPastAppointments = createSelector(
   state => state.appointments.past,
   past => {
-    return past?.filter(isValidPastAppointment).sort(sortByDateDescending);
+    if (!past) {
```

Comment:
Still need to revert this selector back to the way it was

Reply:
🤦 👍